project.html

```diff
@@ -293,11 +308,10 @@
   (i.e., would this be publishable at a good conference)?
 </ul>
 <p>
-Regardless of the group size, all groups must do the same basic amount of work
```

Comment:
Did you mean to delete this part? It seems still relevant.

Reply:
Contradicts another sentence.